src/olympia/migrations/954-add-file-is-restart-required.sql

```diff
@@ -0,0 +1,5 @@
+-- is_restart_required replaces the old no_restart field. It's the exact opposite.
+ALTER TABLE `files`
+    ADD COLUMN `is_restart_required` bool NOT NULL,
+    MODIFY `no_restart` bool;  -- Allow nulls to prepare for column removal next push.
+UPDATE `files` SET `is_restart_required` = NOT `no_restart`;
```

Comment:
I wonder if ops is going to kill us for this. Might be worth giving them a heads-up and check timing on dev and stage.